packages/testsuite/cypress/support/form-editing.ts

```diff
@@ -1,7 +1,23 @@
 Cypress.Commands.add("editForm", (formId) => {
-  const editButton = "#" + formId + ' a.clickable[data-operation="edit"';
+  const editButton = "#" + formId + ' a.clickable[data-operation="edit"]';
   cy.get(`#${formId}-editing`).should("not.be.visible");
   cy.get(editButton).click();
+  // Workauround - sometimes is required to click more times, especialy on overloaded system.
+  // The while cycle is alwas 5 times repeated, but on edit button is clicked only when is visible
+  let reClickCount = 0;
+  while (reClickCount < 5) {
```

Comment:
`for (let i = 0; i < 5; i++)` would be more suitable.

Reply:
For cycle used. The promise is executed in another thread so you can't set `reClickCount` to 5 for example because it isn't thread safe operation.